integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/database/RedbeamsDatabaseTestDto.java

```diff
@@ -0,0 +1,153 @@
+    }
+
+    public RedbeamsDatabaseTestDto withRequest(DatabaseV4Request request) {
+        setRequest(request);
+        return this;
+    }
+
+    public RedbeamsDatabaseTestDto withName(String name) {
+        getRequest().setName(name);
+        setName(name);
+        return this;
+    }
+
+    public RedbeamsDatabaseTestDto withDescription(String description) {
+        getRequest().setDescription(description);
+        return this;
+    }
+
+    public RedbeamsDatabaseTestDto withConnectionPassword(String password) {
+        getRequest().setConnectionPassword(password);
+        return this;
+    }
+
+    public RedbeamsDatabaseTestDto withConnectionUserName(String username) {
+        getRequest().setConnectionUserName(username);
+        return this;
+    }
+
+    public RedbeamsDatabaseTestDto withConnectionURL(String connectionURL) {
+        getRequest().setConnectionURL(connectionURL);
+        return this;
+    }
+
+    public RedbeamsDatabaseTestDto withType(String type) {
+        getRequest().setType(type);
+        return this;
+    }
+
+    public RedbeamsDatabaseTestDto withEnvironmentCrn(String environmentCrn) {
+        getRequest().setEnvironmentCrn(environmentCrn);
+        return this;
+    }
+
+    @Override
+    public List<DatabaseV4Response> getAll(RedbeamsClient client) {
+        DatabaseV4Endpoint databaseV4Endpoint = client.getEndpoints().databaseV4Endpoint();
+        return databaseV4Endpoint.list(client.getEnvironmentCrn()).getResponses().stream()
+                .filter(s -> s.getName() != null)
+                .collect(Collectors.toList());
+    }
+
+    @Override
+    protected String name(DatabaseV4Response entity) {
+        return entity.getName();
+    }
+
+    @Override
+    public void delete(TestContext testContext, DatabaseV4Response entity, RedbeamsClient client) {
+        try {
+            client.getEndpoints().databaseV4Endpoint().deleteByName(client.getEnvironmentCrn(), entity.getName());
+        } catch (Exception e) {
+            LOGGER.warn("Something went wrong on {} purge. {}", entity.getName(), ResponseUtil.getErrorMessage(e), e);
+        }
+    }
+
+    @Override
+    public int order() {
+        return 500;
+    }
+
+    // private static Function<IntegrationTestContext, RedbeamsDatabaseTestDto> getTestContext(String key) {
+    //     return testContext -> testContext.getContextParam(key, RedbeamsDatabaseTestDto.class);
+    // }
+
+    // static Function<IntegrationTestContext, RedbeamsDatabaseTestDto> getNew() {
+    //     return testContext -> new RedbeamsDatabaseTestDto();
+    // }
+
+    // public static RedbeamsDatabaseTestDto request() {
+    //     return new RedbeamsDatabaseTestDto();
+    // }
+
+    // public static Assertion<RedbeamsDatabaseTestDto> assertThis(BiConsumer<RedbeamsDatabaseTestDto, IntegrationTestContext> check) {
+    //     return new Assertion<>(getTestContext(GherkinTest.RESULT), check);
+    // }
```

Comment:
...wondering again as above

Reply:
This stuff can go, I think. Will remove.